pyproject.toml

```diff
@@ -183,6 +183,7 @@ ignore = [  # NOTE: non-permeanent exclusions should be added to `.ruff.toml` in
     "E402",  # Module level import not at top of file
     "PGH001",  # No builtin eval() allowed
     "S101",  # Use of assert detected
+    "RUF015",  # Prefer next({iterable}) over single element slice
```

Comment:
I see no need for ignoring this.

Reply:
Let's move this discussion to a new Issue? Ignoring it for now is the option with the least number of changes.
I'll move the ignore to `.ruff.toml`, indicating it's not necessarily permanent.

Reply:
I made #15145

Reply:
Also, RUF015 ignore has been moved to `.ruff.toml`.